lib/helpers.coffee

```diff
@@ -4,6 +4,12 @@ path = require 'path'
 child_process = require 'child_process'
 
 Helpers = module.exports =
+  shouldTriggerLinter: (linter, wasTriggeredOnChange, scopes)->
+    # Trigger fly linters on save, but not save linters on fly
```

Comment:
Sorry, I misread the comment thinking fly linters would *not* trigger on save. It's only the reverse.

CoffeeLint will lint on the fly, but there are times (often when editing the config file) when I save specifically to trigger the linter.